resources/styles.css

```diff
@@ -250,7 +280,6 @@ main {
     bottom: 56px;
     left: 0;
     right: 0;
-    overflow: auto;
```

Comment:
Removing this breaks the behavour of the bottom bar.

Please, make sure you properly test any change made to the css

Reply:
<img width="219" alt="screen shot 2017-11-01 at 10 32 10 am" src="https://user-images.githubusercontent.com/347483/32279720-f1608efa-beef-11e7-8c46-38ddf1bb891b.png">

If `main` is set to `overflow: auto`, the top bar overlaps the tooltip.